fastapi/dependencies/utils.py

```diff
@@ -109,20 +109,44 @@ def get_param_sub_dependant(
     security_scopes: Optional[List[str]] = None,
 ) -> Dependant:
     assert depends.dependency
-    return get_sub_dependant(
+    dependant = get_sub_dependant(
         depends=depends,
         dependency=depends.dependency,
         path=path,
         name=param_name,
         security_scopes=security_scopes,
     )
+    for query_param in dependant.query_params:
+        query_param_field = depends.dependency.model_fields.get(query_param.name)
+        if query_param_field:
+            query_param.field_info.description = (
+                query_param_field.description or query_param_field.title or ""
+            )
+        else:
+            query_param.field_info.description = (
+                query_param_field.description or query_param_field.title or ""
+            )
```

Comment:
Looks like the same actions are performed regardless of the condition. Here's a more concise version:
```python
for query_param in dependant.query_params:
    query_param.field_info.description = query_param_field.description or query_param_field.title or ""
```
This should yield the same result.

Do you perhaps mean the following?
```python
for query_param in dependant.query_params:
    if not query_param.field_info.description:
        query_param.field_info.description = query_param_field.description or query_param_field.title or ""
```
to fill description if it empty.

Reply:
Okay, thank you for the suggestion